UnityProject/Assets/Scripts/Managers/GameManager.cs

```diff
@@ -50,25 +50,32 @@ private void OnValidate()
 		}
 	}
 
-	private void OnLevelFinishedLoading(Scene scene, LoadSceneMode mode)
-	{
-		if (scene.name == "DeathMatch")
-		{
-			counting = true;
-		}
-	}
-
-	public void SyncTime(float currentTime)
-	{
-		GetRoundTime = currentTime;
-	}
-
-	public void ResetRoundTime()
-	{
-		GetRoundTime = cacheTime;
-		restartTime = 10f;
-		counting = true;
-	}
+    private void OnLevelFinishedLoading(Scene scene, LoadSceneMode mode)
+    {
+        if (scene.name == "DeathMatch" || scene.name == "OutpostDeathmatch")
```

Comment:
We could just remove it, as we are only working on TDM and Deathmatch gamemodes till we move the whole repo to round anyway.

Reply:
yeah but it could cause unexpected behaviors on other scenes. I think we should remove it when we have the proper round system in.